test/fixtures/cli/Dockerfile

```
@@ -1,5 +1,5 @@
 ARG BRANCH_NAME
-FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner:${BRANCH_NAME}-ci
+FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner:${BRANCH_NAME}-ci-ruby-2.7
```

Should this use `RUBY_VERSION` like the others?

```suggestion
ARG RUBY_VERSION
FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner:${BRANCH_NAME}-ci-ruby-${RUBY_VERSION}
```

---

Absolutely, I was getting confused with the `cli` Docker image itself (but even then burying it in the Dockerfile would have been bad).  Now fixed - including passing of the argument through `docker-compose.yml`.